model:
  type: 1 #{0,1,2} -> {RRAANSAC, LMedS, RMSAC}
  distance_thresh: 0.03 # distance to model threshold
